feat(replay): Add troubleshooting section for "Replay does not look correct" (#11375)
getsentry/sentry-docs: docs/platforms/javascript/common/session-replay/troubleshooting.mdx

description: "Troubleshooting Session Replay-specific Issues"
---

> <Expandable title="Replay does not look correct" permalink>

Reviewer: Maybe this could be more specific? Like "Replay does not match my site" (or application)

Author: ```suggestion
<Expandable title="Replay does not match my application" permalink>
```

> There are a few category of cases where the replay playback does not match how your application looks. Below are some common causes and what next steps are if you experience this.

Reviewer: ```suggestion
There are a few cases where the replay playback does not match how your application looks. Below are some common causes and what the next steps are if you experience this.
```

> <Expandable title="Replay does not match my application" permalink>
> 
> There are a few cases where the replay playback does not match how your application looks. Below are some common causes and what the next steps are if you experience this.

Reviewer 2: ```suggestion
<Expandable title="Replay doesn't match my application" permalink>

There are a few cases where the replay playback won't match the look of your application. Below, are some common causes and solutions if you experience this.
```

> ### Missing elements from the replay
> 
> * Usage of unsupported and/or deprecated elements such as `<frame>` (not to be confused with `<iframe>`)
> * Element is blocked by the [privacy configuration](/platforms/javascript/session-replay/privacy/#blocking) (by default, elements with the class name `sentry-block` or the attribute `data-sentry-block`)
> 
> ### CSS is wrong or missing
> 
> * There may be edge cases with complex CSS selectors which break the styling. Please [file an issue on GitHub](https://github.com/getsentry/sentry-javascript/issues) if you enounter this.

Reviewer 2: ```suggestion
### Missing elements from the replay

* You may be using unsupported and/or deprecated elements such as `<frame>` (not to be confused with `<iframe>`).
* The element may be blocked by the [privacy configuration](/platforms/javascript/session-replay/privacy/#blocking) (by default, elements with the class name `sentry-block` or the attribute `data-sentry-block`).

### CSS is wrong or missing

* There may be edge cases with complex CSS selectors which break the styling. Please [file an issue on GitHub](https://github.com/getsentry/sentry-javascript/issues) if you encounter this.
```
